_includes/project-links.html

```diff
@@ -0,0 +1,15 @@
+<div class="row mx-1">
+  {% capture acknowledgements %}{{ "/acknowledgements/" | absolute_url }}{% endcapture %}
+  {% capture list_items %}
+  {% include list-group-item.html url="https://github.com/kroxylicious" content="GitHub Organisation" icon="github" %}
+  {% include list-group-item.html url="http://kroxylicious.io/kroxylicious" content="Kroxylicious Documentation" icon="file-earmark-code" %}
+  {% include list-group-item.html url="https://github.com/kroxylicious/design" content="Design & Discussion" icon="diagram-3" %}
+  {% include list-group-item.html url="https://github.com/kroxylicious/kroxylicious/blob/main/LICENSE" content="License" icon="file-earmark-text" %}
+  {% include list-group-item.html url="https://kroxylicious.slack.com" content="Community & Development chat via Slack" icon="slack" %}
```

Comment:
need to add the slack invite link somewhere, maybe having the invite link as our slack link would be good enough.